src/data/bos-components.ts

```diff
@@ -109,6 +112,9 @@ export const componentsByNetworkId = ((): Record<NetworkId, NetworkComponents |
         indexPage: `${testnetTLA}/widget/Search.IndexPage`,
         typeAheadDropdown: `${testnetTLA}/widget/Search.TypeAheadDropdown`,
       },
+      settings: {
+        index: 'discom.testnet/widget/Settings.Index',
```

Comment:
We should use `${testnetTLA}` for all of our official `near` components in `testnet`. For example:

```
index: `${testnetTLA}/widget/Settings.Index`
```